src/Microsoft.Health.Fhir.Core/Features/Operations/Reindex/ReindexJobTask.cs

```diff
@@ -283,15 +283,34 @@ public async Task ExecuteAsync(ReindexJobRecord reindexJobRecord, WeakETag weakE
                         }
                     }
 
-                    // if our received CancellationToken is cancelled we should
+                    // Get the latest version of the reindex job in case another thread has updated it
```

Comment:
>Get the latest version of the reindex job in case another thread has updated it [](http://example.com/codeflow?start=23&length=79)

I would drop few words why we doing it here and only here considering there is plenty of places where we update job record.

Reply:
I have added a comment.